website/content/en/highlights/2025-02-24-0-45-0-upgrade-guide.md

```diff
@@ -0,0 +1,22 @@
+---
+date: "2025-02-24"
+title: "0.45 Upgrade Guide"
+description: "An upgrade guide that addresses breaking changes in 0.45.0"
+authors: [ "pront" ]
+release: "0.45.0"
+hide_on_release_notes: false
+badges:
+  type: breaking change
+---
+
+VRL version 0.22.0 included a couple of breaking Changes.
+
+## Truncate no longer supports `ellipsis`
+
+The `ellipsis` was deprecated. You can use `suffix: ...` instead.
+
+## `slice` VRL function
+
+The `slice` function now preserves more type information on array elements. This might change its
+fallibility in your VRL programs.
```

Comment:
Maybe we could add an example here of something that used to fail (or not fail) and now does the opposite?